commit 56e08497ff1591d0879f6a9a3b8920a2d42ce1f4
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2019-09-05T08:56:16+01:00 (5 weeks ago)
    
    MANTA-4318 pgstatsmon needs to be able to talk to buckets postgres

diff --git a/config/services/pgstatsmon/service.json b/config/services/pgstatsmon/service.json
index 4c1bb2f..def4341 100644
--- a/config/services/pgstatsmon/service.json
+++ b/config/services/pgstatsmon/service.json
@@ -6,9 +6,17 @@
 	"metadata": {
 		"DATABASE_USER": "pgstatsmon",
 		"VM_TAG_NAME": "manta_role",
-		"VM_TAG_VALUE": "postgres",
+		"VM_TAG_VALUE": "*postgres",
 		"NIC_TAG": "manta",
 		"VMAPI_POLL_INTERVAL": "60000",
-		"SCRAPE_INTERVAL": "60000"
+		"SCRAPE_INTERVAL": "60000",
+		"VM_ROLES": [ {
+			"name": "postgres",
+			"database": "moray"
+		}, {
+			"name": "buckets-postgres",
+			"database": "boray",
+			"last": true
+		} ]
 	}
 }
